overloaded-virtual warning

```
In file included from /usr/include/gtkmm-3.0/gtkmm/treeview.h:34,
                 from maintreeview.h:16,
                 from mainwindow.h:13,
                 from main.cc:12:
/usr/include/gtkmm-3.0/gtkmm/container.h:133:16: error: ‘virtual void Gtk::Container::on_remove(Gtk::Widget*)’ was hidden [-Werror=overloaded-virtual=]
  133 |   virtual void on_remove(Widget* widget);
      |                ^~~~~~~~~
mainwindow.h:41:14: note:   by ‘void Grop::MainWindow::on_remove()’
   41 |         void on_remove();
      |              ^~~~~~~~~
```

--- a/grop/mainwindow.cc
+++ b/grop/mainwindow.cc
@@ -115,7 +115,7 @@ void MainWindow::build_menu_bar()
 	m_action_group->add(Action::create("MenuPackage", "_Package"));
 	m_action_group->add(m_action_properties, mem_fun(this, &MainWindow::on_properties));
 	m_action_group->add(m_action_porgball, mem_fun(this, &MainWindow::on_porgball));
-	m_action_group->add(m_action_remove, mem_fun(this, &MainWindow::on_remove));
+	m_action_group->add(m_action_remove, mem_fun(this, &MainWindow::on_remove_pkg));
 	m_action_group->add(m_action_unlog, mem_fun(*this, &MainWindow::on_unlog));
 
 	m_action_group->add(Action::create("MenuHelp", "_Help"));
@@ -288,7 +288,7 @@ void MainWindow::on_unlog()
 }
 
 
-void MainWindow::on_remove()
+void MainWindow::on_remove_pkg()
 {
 	g_return_if_fail(Opt::logdir_writable() && m_selected_pkg != NULL);
 

--- a/grop/mainwindow.h
+++ b/grop/mainwindow.h
@@ -38,7 +38,7 @@ class MainWindow : public Gtk::Window
 	void on_pkg_menu();
 	void on_properties();
 	void on_unlog();
-	void on_remove();
+	void on_remove_pkg();
 	void on_porgball();
 	void on_popup_menu(GdkEventButton*);
 	void on_2button_press(GdkEventButton*);

